### ISSUE TITLE
regression? g8Test has no effect in sbt-giter8 0.8.0

### ISSUE THREAD

**Issue by xuwei-k** [foundweekends/giter8#292]

**Comment by xuwei-k** [comment]
I believe regression in https://github.com/foundweekends/giter8/pull/277

The`g8Test` task should not only apply template but also execute `sbt test` 

https://github.com/foundweekends/giter8/blob/v0.8.0/plugin/src/main/scala/giter8/Giter8Plugin.scala

> inputKey[Unit]("Run `sbt test` in output to smoke-test the templates")

/cc @foxmk

because before #277 

https://travis-ci.org/foundweekends/giter8/builds/215623566#L1784

```
Running giter8 / simple
[error] Java HotSpot(TM) 64-Bit Server VM warning: ignoring option MaxPermSize=256m; support was removed in 8.0
[info] Getting org.scala-sbt sbt 0.13.13 ...
[info] :: retrieving :: org.scala-sbt#boot-app
[info] 	confs: [default]
[info] 	50 artifacts copied, 0 already retrieved (17645kB/61ms)
[info] Getting Scala 2.10.6 (for sbt)...
[info] :: retrieving :: org.scala-sbt#boot-scala
[info] 	confs: [default]
[info] 	5 artifacts copied, 0 already retrieved (24494kB/33ms)
[info] [info] Loading project definition from /tmp/sbt_a6a30f6/simple/project
[info] [info] Updating {file:/tmp/sbt_a6a30f6/simple/project/}simple-build...
[info] [info] Resolving org.foundweekends.giter8#sbt-giter8;0.7.3-SNAPSHOT ...
[info] 
[info] [info] Resolving org.foundweekends.giter8#sbt-giter8;0.7.3-SNAPSHOT ...
[info] 
[info] [info] Resolving org.foundweekends.giter8#giter8-lib_2.10;0.7.3-SNAPSHOT ...
[info] 
[info] [info] Resolving org.foundweekends.giter8#giter8-lib_2.10;0.7.3-SNAPSHOT ...
[info] 
[info] [info] Resolving org.scala-lang#scala-library;2.10.6 ...
[info] 
[info] [info] Resolving org.clapper#scalasti_2.10;2.1.2 ...
[info] 
[info] [info] Resolving org.clapper#grizzled-scala_2.10;3.1.0 ...
[info] 
[info] [info] Resolving org.clapper#classutil_2.10;1.1.0 ...
[info] 
[info] [info] Resolving org.ow2.asm#asm;5.1 ...
[info] 
[info] [info] Resolving org.ow2.asm#asm-commons;5.1 ...
[info] 
[info] [info] Resolving org.ow2.asm#asm-tree;5.1 ...
[info] 
[info] [info] Resolving org.ow2.asm#asm-util;5.1 ...
[info] 
[info] [info] Resolving org.antlr#ST4;4.0.8 ...
[info] 
[info] [info] Resolving org.antlr#antlr-runtime;3.5.2 ...
[info] 
[info] [info] Resolving org.scala-lang#scala-reflect;2.10.6 ...
[info] 
[info] [info] Resolving org.eclipse.jgit#org.eclipse.jgit.pgm;3.7.0.201502260915-r ...
[info] 
[info] [info] Resolving args4j#args4j;2.0.12 ...
[info] 
[info] [info] Resolving org.apache.commons#commons-compress;1.6 ...
[info] 
[info] [info] Resolving org.tukaani#xz;1.4 ...
[info] 
[info] [info] Resolving org.eclipse.jgit#org.eclipse.jgit.archive;3.7.0.201502260915-r ...
[info] 
[info] [info] Resolving org.eclipse.jgit#org.eclipse.jgit;3.7.0.201502260915-r ...
[info] 
[info] [info] Resolving com.jcraft#jsch;0.1.50 ...
[info] 
[info] [info] Resolving com.googlecode.javaewah#JavaEWAH;0.7.9 ...
[info] 
[info] [info] Resolving org.apache.httpcomponents#httpclient;4.1.3 ...
[info] 
[info] [info] Resolving org.apache.httpcomponents#httpcore;4.1.4 ...
[info] 
[info] [info] Resolving commons-logging#commons-logging;1.1.1 ...
[info] 
[info] [info] Resolving commons-codec#commons-codec;1.4 ...
[info] 
[info] [info] Resolving org.slf4j#slf4j-api;1.7.2 ...
[info] 
[info] [info] Resolving org.osgi#org.osgi.core;4.3.1 ...
[info] 
[info] [info] Resolving org.eclipse.jgit#org.eclipse.jgit.ui;3.7.0.201502260915-r ...
[info] 
[info] [info] Resolving commons-io#commons-io;2.4 ...
[info] 
[info] [info] Resolving org.codehaus.plexus#plexus-archiver;2.7.1 ...
[info] 
[info] [info] Resolving org.codehaus.plexus#plexus-container-default;1.0-alpha-9-stable-1 ...
[info] 
[info] [info] Resolving org.codehaus.plexus#plexus-utils;3.0.18 ...
[info] 
[info] [info] Resolving org.codehaus.plexus#plexus-io;2.2 ...
[info] 
[info] [info] Resolving org.scala-sbt#scripted-plugin;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#sbt;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#main;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#actions;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#classpath;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-lang#scala-compiler;2.10.6 ...
[info] 
[info] [info] Resolving org.scala-sbt#interface;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#io;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#control;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#launcher-interface;1.0.0-M1 ...
[info] 
[info] [info] Resolving org.scala-sbt#completion;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#collections;0.13.13 ...
[info] 
[info] [info] Resolving jline#jline;2.13 ...
[info] 
[info] [info] Resolving org.fusesource.jansi#jansi;1.11 ...
[info] 
[info] [info] Resolving org.scala-sbt#api;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#classfile;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#logging;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#process;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#compiler-integration;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#incremental-compiler;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#relation;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#compile;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#persist;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-tools.sbinary#sbinary_2.10;0.4.2 ...
[info] 
[info] [info] Resolving org.scala-sbt#compiler-ivy-integration;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#ivy;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#cross;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt.ivy#ivy;2.3.0-sbt-2cf13e211b2cb31f0d3b317289dca70eca3362f6 ...
[info] 
[info] [info] Resolving org.scala-sbt#serialization_2.10;0.1.2 ...
[info] 
[info] [info] Resolving org.scala-lang.modules#scala-pickling_2.10;0.10.1 ...
[info] 
[info] [info] Resolving org.scalamacros#quasiquotes_2.10;2.0.1 ...
[info] 
[info] [info] Resolving org.json4s#json4s-core_2.10;3.2.10 ...
[info] 
[info] [info] Resolving org.json4s#json4s-ast_2.10;3.2.10 ...
[info] 
[info] [info] Resolving com.thoughtworks.paranamer#paranamer;2.6 ...
[info] 
[info] [info] Resolving org.spire-math#jawn-parser_2.10;0.6.0 ...
[info] 
[info] [info] Resolving org.spire-math#json4s-support_2.10;0.6.0 ...
[info] 
[info] [info] Resolving org.scala-sbt#run;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#task-system;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#tasks;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#tracking;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#cache;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#testing;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#test-agent;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#test-interface;1.0 ...
[info] 
[info] [info] Resolving org.scala-sbt#main-settings;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#apply-macro;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#command;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#template-resolver;0.1 ...
[info] 
[info] [info] Resolving org.scala-sbt#logic;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#compiler-interface;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-lang#jline;2.10.6 ...
[info] 
[info] [info] Resolving org.fusesource.jansi#jansi;1.4 ...
[info] [info] Done updating.
[info] [info] Set current project to simple (in build file:/tmp/sbt_a6a30f6/simple/)
[info] [info] ()
[info] [success] Total time: 1 s, completed Mar 27, 2017 5:37:51 PM
[info] [info] Updating {file:/tmp/sbt_a6a30f6/simple/}simple...
[info] [info] Resolving org.scala-lang#scala-library;2.10.6 ...
[info] 
[info] [info] Resolving org.scala-lang#scala-compiler;2.10.6 ...
[info] 
[info] [info] Resolving org.scala-lang#scala-reflect;2.10.6 ...
[info] 
[info] [info] Resolving org.scala-lang#jline;2.10.6 ...
[info] 
[info] [info] Resolving org.fusesource.jansi#jansi;1.4 ...
[info] 
[info] [info] Resolving org.scala-sbt#scripted-sbt;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#io;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#control;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#logging;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#interface;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#process;0.13.13 ...
[info] 
[info] [info] Resolving jline#jline;2.13 ...
[info] 
[info] [info] Resolving org.fusesource.jansi#jansi;1.11 ...
[info] 
[info] [info] Resolving org.scala-sbt#scripted-framework;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#sbt-launch;0.13.13 ...
[info] [info] Done updating.
[info] Running simple / scripted
[info] [success] Total time: 39 s, completed Mar 27, 2017 5:38:30 PM
[info] [success] Total time: 0 s, completed Mar 27, 2017 5:38:30 PM
[info] Running simple / scripted
[info] [error] Java HotSpot(TM) 64-Bit Server VM warning: ignoring option MaxPermSize=256m; support was removed in 8.0
[info] [info] Getting org.scala-sbt sbt 0.13.13 ...
[info] [info] :: retrieving :: org.scala-sbt#boot-app
[info] [info] 	confs: [default]
[info] [info] 	50 artifacts copied, 0 already retrieved (17645kB/77ms)
[info] [info] Getting Scala 2.10.6 (for sbt)...
[info] [info] :: retrieving :: org.scala-sbt#boot-scala
[info] [info] 	confs: [default]
[info] [info] 	5 artifacts copied, 0 already retrieved (24494kB/42ms)
[info] [info] [info] Set current project to New Project (in build file:/tmp/sbt_21b6d2c/scripted/)
[info] [info] [info] Updating {file:/tmp/sbt_21b6d2c/scripted/}scripted...
[info] [info] [info] Resolving org.scala-lang#scala-library;2.11.2 ...
[info] [info] 
[info] [info] [info] Resolving net.databinder#unfiltered_2.11;0.9.0-beta1 ...
[info] [info] 
[info] [info] [info] Resolving net.databinder#unfiltered-util_2.11;0.9.0-beta1 ...
[info] [info] 
[info] [info] [info] Resolving org.scala-lang.modules#scala-xml_2.11;1.0.1 ...
[info] [info] 
[info] [info] [info] Resolving commons-codec#commons-codec;1.4 ...
[info] [info] 
[info] [info] [info] Resolving net.databinder.dispatch#dispatch-core_2.11;0.12.0 ...
[info] [info] 
[info] [info] [info] Resolving org.scala-lang.modules#scala-xml_2.11;1.0.6 ...
[info] [info] 
[info] [info] [info] Resolving com.ning#async-http-client;1.9.11 ...
[info] [info] 
[info] [info] [info] Resolving io.netty#netty;3.10.0.Final ...
[info] [info] 
[info] [info] [info] Resolving org.slf4j#slf4j-api;1.7.7 ...
[info] [info] 
[info] [info] [info] Resolving org.scala-lang#scala-compiler;2.11.2 ...
[info] [info] 
[info] [info] [info] Resolving org.scala-lang#scala-reflect;2.11.2 ...
[info] [info] 
[info] [info] [info] Resolving org.scala-lang.modules#scala-xml_2.11;1.0.2 ...
[info] [info] 
[info] [info] [info] Resolving org.scala-lang.modules#scala-parser-combinators_2.11;1.0.2 ...
[info] [info] 
[info] [info] [info] Resolving jline#jline;2.12 ...
[info] [info] [info] Done updating.
[info] [info] [info] Compiling 2 Scala sources to /tmp/sbt_21b6d2c/scripted/target/scala-2.11/classes...
[info] [info] [info] Compiling 1 Scala source to /tmp/sbt_21b6d2c/scripted/target/scala-2.11/test-classes...
[info] [info] [error] /tmp/sbt_21b6d2c/scripted/src/test/scala/invalid.scala:1: expected class or object definition
[info] [info] [error] invalid file
[info] [info] [error] ^
[info] [info] [error] one error found
[info] [info] [error] (test:compileIncremental) Compilation failed
[info] [info] [error] Total time: 6 s, completed Mar 27, 2017 5:38:46 PM
[info] [error] x simple / scripted
[info] [error]    {line 1}  Command failed: test failed
[info] java.lang.RuntimeException: Failed tests:
[info] 	simple / scripted
[info] 
[info] 	at scala.sys.package$.error(package.scala:27)
[info] 	at sbt.test.ScriptedRunner.runAll(ScriptedTests.scala:156)
[info] 	at sbt.test.ScriptedRunner.run(ScriptedTests.scala:151)
[info] 	at sbt.test.ScriptedRunner.run(ScriptedTests.scala:125)
[info] 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[info] 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[info] 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[info] 	at java.lang.reflect.Method.invoke(Method.java:483)
[info] 	at sbt.ScriptedPlugin$$anonfun$scriptedTask$1$$anonfun$apply$3$$anonfun$apply$4.apply(ScriptedPlugin.scala:61)
[info] 	at sbt.ScriptedPlugin$$anonfun$scriptedTask$1$$anonfun$apply$3$$anonfun$apply$4.apply(ScriptedPlugin.scala:57)
[info] 	at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47)
[info] 	at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:40)
[info] 	at sbt.std.Transform$$anon$4.work(System.scala:63)
[info] 	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
[info] 	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
[info] 	at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:17)
[info] 	at sbt.Execute.work(Execute.scala:237)
[info] 	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
[info] 	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
[info] 	at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:159)
[info] 	at sbt.CompletionService$$anon$2.call(CompletionService.scala:28)
[info] 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[info] 	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
[info] 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[info] 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
[info] 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
[info] 	at java.lang.Thread.run(Thread.java:745)
[info] [error] (test:g8Test) Failed tests:
[info] [error] 	simple / scripted
[info] [error] Total time: 16 s, completed Mar 27, 2017 5:38:46 PM
[info] + giter8 / simple
[success] Total time: 106 s, completed Mar 27, 2017 5:38:48 PM
```

after  #277 

https://travis-ci.org/foundweekends/giter8/builds/215775918#L1987

```
Running giter8 / simple
[error] Java HotSpot(TM) 64-Bit Server VM warning: ignoring option MaxPermSize=256m; support was removed in 8.0
[info] Getting org.scala-sbt sbt 0.13.13 ...
[info] :: retrieving :: org.scala-sbt#boot-app
[info] 	confs: [default]
[info] 	50 artifacts copied, 0 already retrieved (17645kB/149ms)
[info] Getting Scala 2.10.6 (for sbt)...
[info] :: retrieving :: org.scala-sbt#boot-scala
[info] 	confs: [default]
[info] 	5 artifacts copied, 0 already retrieved (24494kB/51ms)
[info] [info] Loading project definition from /tmp/sbt_d86e8b9c/simple/project
[info] [info] Updating {file:/tmp/sbt_d86e8b9c/simple/project/}simple-build...
[info] [info] Resolving org.foundweekends.giter8#sbt-giter8;0.7.3-SNAPSHOT ...
[info] 
[info] [info] Resolving org.foundweekends.giter8#sbt-giter8;0.7.3-SNAPSHOT ...
[info] 
[info] [info] Resolving org.foundweekends.giter8#giter8-library_2.10;0.7.3-SNAPSHOT ...
[info] 
[info] [info] Resolving org.foundweekends.giter8#giter8-library_2.10;0.7.3-SNAPSHOT ...
[info] 
[info] [info] Resolving org.scala-lang#scala-library;2.10.6 ...
[info] 
[info] [info] Resolving com.google.code.findbugs#jsr305;3.0.1 ...
[info] 
[info] [info] Resolving org.clapper#scalasti_2.10;2.1.2 ...
[info] 
[info] [info] Resolving org.clapper#grizzled-scala_2.10;3.1.0 ...
[info] 
[info] [info] Resolving org.clapper#classutil_2.10;1.1.0 ...
[info] 
[info] [info] Resolving org.ow2.asm#asm;5.1 ...
[info] 
[info] [info] Resolving org.ow2.asm#asm-commons;5.1 ...
[info] 
[info] [info] Resolving org.ow2.asm#asm-tree;5.1 ...
[info] 
[info] [info] Resolving org.ow2.asm#asm-util;5.1 ...
[info] 
[info] [info] Resolving org.antlr#ST4;4.0.8 ...
[info] 
[info] [info] Resolving org.antlr#antlr-runtime;3.5.2 ...
[info] 
[info] [info] Resolving org.scala-lang#scala-reflect;2.10.6 ...
[info] 
[info] [info] Resolving org.eclipse.jgit#org.eclipse.jgit.pgm;3.7.0.201502260915-r ...
[info] 
[info] [info] Resolving args4j#args4j;2.0.12 ...
[info] 
[info] [info] Resolving org.apache.commons#commons-compress;1.6 ...
[info] 
[info] [info] Resolving org.tukaani#xz;1.4 ...
[info] 
[info] [info] Resolving org.eclipse.jgit#org.eclipse.jgit.archive;3.7.0.201502260915-r ...
[info] 
[info] [info] Resolving org.eclipse.jgit#org.eclipse.jgit;3.7.0.201502260915-r ...
[info] 
[info] [info] Resolving com.jcraft#jsch;0.1.50 ...
[info] 
[info] [info] Resolving com.googlecode.javaewah#JavaEWAH;0.7.9 ...
[info] 
[info] [info] Resolving org.apache.httpcomponents#httpclient;4.1.3 ...
[info] 
[info] [info] Resolving org.apache.httpcomponents#httpcore;4.1.4 ...
[info] 
[info] [info] Resolving commons-logging#commons-logging;1.1.1 ...
[info] 
[info] [info] Resolving commons-codec#commons-codec;1.4 ...
[info] 
[info] [info] Resolving org.slf4j#slf4j-api;1.7.2 ...
[info] 
[info] [info] Resolving org.osgi#org.osgi.core;4.3.1 ...
[info] 
[info] [info] Resolving org.eclipse.jgit#org.eclipse.jgit.ui;3.7.0.201502260915-r ...
[info] 
[info] [info] Resolving commons-io#commons-io;2.4 ...
[info] 
[info] [info] Resolving org.codehaus.plexus#plexus-archiver;2.7.1 ...
[info] 
[info] [info] Resolving org.codehaus.plexus#plexus-container-default;1.0-alpha-9-stable-1 ...
[info] 
[info] [info] Resolving org.codehaus.plexus#plexus-utils;3.0.18 ...
[info] 
[info] [info] Resolving org.codehaus.plexus#plexus-io;2.2 ...
[info] 
[info] [info] Resolving org.scala-sbt#scripted-plugin;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#sbt;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#main;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#actions;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#classpath;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-lang#scala-compiler;2.10.6 ...
[info] 
[info] [info] Resolving org.scala-sbt#interface;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#io;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#control;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#launcher-interface;1.0.0-M1 ...
[info] 
[info] [info] Resolving org.scala-sbt#completion;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#collections;0.13.13 ...
[info] 
[info] [info] Resolving jline#jline;2.13 ...
[info] 
[info] [info] Resolving org.fusesource.jansi#jansi;1.11 ...
[info] 
[info] [info] Resolving org.scala-sbt#api;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#classfile;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#logging;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#process;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#compiler-integration;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#incremental-compiler;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#relation;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#compile;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#persist;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-tools.sbinary#sbinary_2.10;0.4.2 ...
[info] 
[info] [info] Resolving org.scala-sbt#compiler-ivy-integration;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#ivy;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#cross;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt.ivy#ivy;2.3.0-sbt-2cf13e211b2cb31f0d3b317289dca70eca3362f6 ...
[info] 
[info] [info] Resolving org.scala-sbt#serialization_2.10;0.1.2 ...
[info] 
[info] [info] Resolving org.scala-lang.modules#scala-pickling_2.10;0.10.1 ...
[info] 
[info] [info] Resolving org.scalamacros#quasiquotes_2.10;2.0.1 ...
[info] 
[info] [info] Resolving org.json4s#json4s-core_2.10;3.2.10 ...
[info] 
[info] [info] Resolving org.json4s#json4s-ast_2.10;3.2.10 ...
[info] 
[info] [info] Resolving com.thoughtworks.paranamer#paranamer;2.6 ...
[info] 
[info] [info] Resolving org.spire-math#jawn-parser_2.10;0.6.0 ...
[info] 
[info] [info] Resolving org.spire-math#json4s-support_2.10;0.6.0 ...
[info] 
[info] [info] Resolving org.scala-sbt#run;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#task-system;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#tasks;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#tracking;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#cache;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#testing;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#test-agent;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#test-interface;1.0 ...
[info] 
[info] [info] Resolving org.scala-sbt#main-settings;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#apply-macro;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#command;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#template-resolver;0.1 ...
[info] 
[info] [info] Resolving org.scala-sbt#logic;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#compiler-interface;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-lang#jline;2.10.6 ...
[info] 
[info] [info] Resolving org.fusesource.jansi#jansi;1.4 ...
[info] [info] Done updating.
[info] [info] Set current project to simple (in build file:/tmp/sbt_d86e8b9c/simple/)
[info] [info] Updating {file:/tmp/sbt_d86e8b9c/simple/}simple...
[info] [info] Packaging /tmp/sbt_d86e8b9c/simple/target/scala-2.10/simple_2.10-0.1-SNAPSHOT-sources.jar ...
[info] [info] Done packaging.
[info] [info] Wrote /tmp/sbt_d86e8b9c/simple/target/scala-2.10/simple_2.10-0.1-SNAPSHOT.pom
[info] [info] Resolving org.scala-lang#scala-library;2.10.6 ...
[info] 
[info] [info] Resolving org.scala-lang#scala-compiler;2.10.6 ...
[info] 
[info] [info] Resolving org.scala-lang#scala-reflect;2.10.6 ...
[info] 
[info] [info] Resolving org.scala-lang#jline;2.10.6 ...
[info] 
[info] [info] Resolving org.fusesource.jansi#jansi;1.4 ...
[info] 
[info] [info] Resolving org.scala-sbt#scripted-sbt;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#io;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#control;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#logging;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#interface;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#process;0.13.13 ...
[info] 
[info] [info] Resolving jline#jline;2.13 ...
[info] 
[info] [info] Resolving org.fusesource.jansi#jansi;1.11 ...
[info] 
[info] [info] Resolving org.scala-sbt#scripted-framework;0.13.13 ...
[info] 
[info] [info] Resolving org.scala-sbt#sbt-launch;0.13.13 ...
[info] [info] Done updating.
[info] [info] :: delivering :: default#simple_2.10;0.1-SNAPSHOT :: 0.1-SNAPSHOT :: integration :: Tue Mar 28 02:36:22 UTC 2017
[info] [info] 	delivering ivy file to /tmp/sbt_d86e8b9c/simple/target/scala-2.10/ivy-0.1-SNAPSHOT.xml
[info] [info] Packaging /tmp/sbt_d86e8b9c/simple/target/scala-2.10/simple_2.10-0.1-SNAPSHOT-javadoc.jar ...
[info] [info] Done packaging.
[info] [info] Packaging /tmp/sbt_d86e8b9c/simple/target/scala-2.10/simple_2.10-0.1-SNAPSHOT.jar ...
[info] [info] Done packaging.
[info] [info] 	published simple_2.10 to /home/travis/.ivy2/local/default/simple_2.10/0.1-SNAPSHOT/poms/simple_2.10.pom
[info] [info] 	published simple_2.10 to /home/travis/.ivy2/local/default/simple_2.10/0.1-SNAPSHOT/jars/simple_2.10.jar
[info] [info] 	published simple_2.10 to /home/travis/.ivy2/local/default/simple_2.10/0.1-SNAPSHOT/srcs/simple_2.10-sources.jar
[info] [info] 	published simple_2.10 to /home/travis/.ivy2/local/default/simple_2.10/0.1-SNAPSHOT/docs/simple_2.10-javadoc.jar
[info] [info] 	published ivy to /home/travis/.ivy2/local/default/simple_2.10/0.1-SNAPSHOT/ivys/ivy.xml
[info] [info] ()
[info] [success] Total time: 1 s, completed Mar 28, 2017 2:36:22 AM
[info] Template applied
[info] [success] Total time: 2 s, completed Mar 28, 2017 2:36:24 AM
[info] [success] Total time: 0 s, completed Mar 28, 2017 2:36:24 AM
[info] Template applied
[info] [success] Total time: 1 s, completed Mar 28, 2017 2:36:24 AM
[info] + giter8 / simple
[success] Total time: 135 s, completed Mar 28, 2017 2:36:27 AM
```

**Comment by eed3si9n** [comment]
This is fixed in 0.9.x branch by rolling back 0.8.0 changes - https://travis-ci.org/foundweekends/giter8/builds/248617633

```
[info] [info] [error] /tmp/sbt_74c5f074/scripted/src/test/scala/invalid.scala:1: expected class or object definition
[info] [info] [error] invalid file
[info] [info] [error] ^
[info] [info] [error] one error found
[info] [info] [error] (test:compileIncremental) Compilation failed
[info] [info] [error] Total time: 4 s, completed Jun 30, 2017 2:13:01 AM
[info] [error] x simple / scripted
[info] [error]    {line 1}  Command failed: test failed
[info] java.lang.RuntimeException: Failed tests:
[info] 	simple / scripted
[info] 
[info] 	at scala.sys.package$.error(package.scala:27)
[info] 	at sbt.test.ScriptedRunner.runAll(ScriptedTests.scala:156)
[info] 	at sbt.test.ScriptedRunner.run(ScriptedTests.scala:151)
[info] 	at sbt.test.ScriptedRunner.run(ScriptedTests.scala:125)
[info] 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[info] 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[info] 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[info] 	at java.lang.reflect.Method.invoke(Method.java:483)
[info] 	at sbt.ScriptedPlugin$$anonfun$scriptedTask$1$$anonfun$apply$6$$anonfun$apply$7.apply(ScriptedPlugin.scala:93)
[info] 	at sbt.ScriptedPlugin$$anonfun$scriptedTask$1$$anonfun$apply$6$$anonfun$apply$7.apply(ScriptedPlugin.scala:89)
[info] 	at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47)
[info] 	at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:40)
[info] 	at sbt.std.Transform$$anon$4.work(System.scala:63)
[info] 	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
[info] 	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
[info] 	at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:17)
[info] 	at sbt.Execute.work(Execute.scala:237)
[info] 	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
[info] 	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
[info] 	at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:159)
[info] 	at sbt.CompletionService$$anon$2.call(CompletionService.scala:28)
[info] 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[info] 	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
[info] 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[info] 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
[info] 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
[info] 	at java.lang.Thread.run(Thread.java:745)
[info] [error] (test:g8Test) Failed tests:
[info] [error] 	simple / scripted
[info] [error] Total time: 14 s, completed Jun 30, 2017 2:13:01 AM
[info] + giter8 / simple
```
